gamemaker.py

```diff
@@ -1,12 +1,31 @@
 class GameMaker:
-  def __init__(self, database):
-    self.database = database
-    self.loggedin = false
-  
-  def login(self, name, password):
-    if (self.database.gameMakerCred = {name, password}):
-      self.loggedin = true
-  //TODO Implement checkStatus
-  def checkStatus(self):
-  //TODO Implement logout
-  def logout(name):
+    def __init__(self, database):
+        self.database = database
+
+    def login(self, name, password):
+        return_value = False
+        current_user = self.database.get_current_user()
+        maker_cred = self.database.get_game_maker_cred()
+        if current_user is None:
+            if maker_cred["maker"] == "maker" and maker_cred["password"] == password:
```

Comment:
Is this hard coding the maker and password in? Or is this dynamically referencing it?